view/range/index.ts

```diff
@@ -156,7 +159,7 @@ function setRangeColor(): void {
 
 visible.subscribe(({ real, fallback }) => {
   setRangeColor()
-  let parsed = parse(real || fallback)
+  let parsed = parse(real || fallback) as Color // WARN: source of potential issues 'parse' function might return an 'undefined'
```

Comment:
Can you wrap `rangeA.style.setProperty` into `if (parsed) {}` to avoid changing the type?

Reply:
Good point! What should be in `else` statement brackets?

Reply:
Nothing